.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: node_js
 
 jobs:
-  fast_finish: true
-  allow_failures:
-    - stage: security scan 🔐
+  # fast_finish: true
+  # allow_failures:
+  #   - stage: security scan 🔐
```

Comment:
Are these lines supposed to be commented out?

Reply:
they can be since the security scan has been commented out for a while.